sdk/core/src/services/UserManagementService.ts

```diff
@@ -377,9 +395,9 @@ export default class UserManagementService {
     }
   }
 
-  private _buildUserAttributes(login: string) {
+  private _buildUserAttributes(login: string, shouldDisableNameNormalisation: boolean) {
```

Comment:
but it is an instance method, it does not need this parameter, it can read it from `this`.